autopush/router/gcm.py

```diff
@@ -84,7 +85,13 @@ def _route(self, notification, router_data):
                 data['enckey'] = notification.headers.get('encryption-key')
             if ('crypto-key' in notification.headers):
                 data['cryptokey'] = notification.headers.get('crypto-key')
-            data['body'] = notification.data
+            udata = urlsafe_b64encode(notification.data)
```

Comment:
Honestly, it was something I was debating. We need to do the base64 decoding for GCM because the library exchanges the data as part of a JSON blob. For APNs, it's a binary exchange to begin with, so data conversion may not be required. Likewise, if we roll out some other bridge, there may be it's own considerations. 

That's why I'm limiting to just GCM for now.